Kernel/API/ToneGenerationPacket.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2023, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Types.h>
+
+struct ToneGenerationPacket {
```

Comment:
:bikeshed:, but maybe `BeepInstruction`? `ToneGenerationPacket` feels both way too verbose and anything with `Tone*` in it feels like a slight mismatch because it will soon no longer be just the tone, but also the length.